libraries/AP_HAL_ChibiOS/hwdef/MatekF405/hwdef.dat

```diff
@@ -168,6 +165,5 @@ define AP_PARAM_MAX_EMBEDDED_PARAM 1024
 
 # save some flash
 include ../include/save_some_flash.inc
```

Comment:
You should remove save_some_flash

Reply:
why? again historically happy with those additional removals

Reply:
As much as possible these should all be consistent if they fit. Having random additions on only some boards hurts users